### PR TITLE
Add Jay Lofstead SC22 episode

### PR DESCRIPTION
**Pull request by mrmundt**
Here is the markdown! Sorry for delaying. I was overly swamped.

@vsoch - I am unsure exactly what you were looking for in terms of "length" - is that file size?

### CODE REVIEW

**Comment by vsoch** [comment]
Yes! Here is an example:
```
$ ls -l rowland-mosbergen/
total 409372
-rw-rw-r-- 1 vanessa vanessa  23915380 Dec 18 19:54 rowland-mosbergen-episode-72.mp3
```
For that you would put the number 23915380 under length.

**Comment by mrmundt** [comment]
@vsoch  - Done! I didn't want to make assumptions about where the file would live, so I did leave `external_media` blank.

**Comment by vsoch** [comment]
Yeah no worries! If the file hasn't changed (it's the one I have) I'll probably just pull your branch here, give it one more final check, and then push. And the audio goes into: https://github.com/usrse/rse-stories-episodes-1. BUT we shouldn't add that until the last minute (I can do it at the time of release!)

**Comment by mrmundt** [comment]
Yup, it's the same one. No changes! So have at it!

File: _posts/2022-05-27-jay-lofstead.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "SC22 Mini-series: Inclusivity Committee"
+author: "@mmundt"
+rse: "Jay Lofstead"
+excerpt: "Jay Lofstead of Sandia National Laboratories has some big plans for the Inclusivity Committee at Supercomputing 2022."
+date: "2022-05-27 8:30:00"
```
Comment (vsoch) [inline _posts/2022-05-27-jay-lofstead.md]: I think next Thursday is the 26th?
```suggestion
date: "2022-05-26 8:30:00"
```

File: _posts/2022-05-27-jay-lofstead.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+length: 16624071
+duration: "00:18:47"
+explicit: "no"
+resources:
+  - name: Supercomputing 2022
+    url: https://sc22.supercomputing.org/
+  - name: SC22 Inclusivity Committee
+    url: https://sc22.supercomputing.org/planning-committee/#Inclusivity
+  - name: SC22 Inclusivity Contact
+    url: sc-inclusivity@info.supercomputing.org
```
Comment (vsoch) [inline _posts/2022-05-27-jay-lofstead.md]: We probably want this to be a mailto: link. You can also just add the email to the episode notes if you want to show (but not link) it.

File: _posts/2022-05-27-jay-lofstead.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Jay Lofstead is a member of technical staff at Sandia National Laboratories
+and this year's Supercomputing 2022 (SC22) Inclusivity Committee Chair. In this
+episode, Jay tells us about his experiences volunteering with SC over the past
+few years, the history of the Inclusivity committee, and his plans for
+implementing change to make Supercomputing 2022 a truly accessible and
+inclusive environment for all participants!
+
+He also ends with a call to action. If you have feedback about your experiences
+at Supercomputing which could help Jay improve inclusivity, please email the
```
Comment (vsoch) [inline _posts/2022-05-27-jay-lofstead.md]:
```suggestion
at Supercomputing that could help Jay improve inclusivity, please email the
```

File: _posts/2022-05-27-jay-lofstead.md (hunk 1 above)
Comment (mrmundt) [reply _posts/2022-05-27-jay-lofstead.md]: You are right - it is. Sorry, I think you said the 27th on Slack but also Thursday and my brain prefers numbers.

**Review by vsoch — APPROVED** [review]

**Pull request merged**